src/Core/src/Handlers/Layout/LayoutHandler.cs

```diff
@@ -23,6 +23,9 @@ public partial class LayoutHandler : ILayoutHandler
 			[nameof(ILayout.ClipsToBounds)] = MapClipsToBounds,
 #if ANDROID || WINDOWS
 			[nameof(IView.InputTransparent)] = MapInputTransparent,
+#endif
+#if GTK
+			[nameof(IView.Visibility)] = MapVisibility,
```

Comment:
The Layout inherit from IView https://github.com/dotnet/maui/blob/main/src/Core/src/Core/ILayout.cs#L9 and the Visibility should be managed by ViewHandler https://github.com/dotnet/maui/blob/main/src/Core/src/Handlers/View/ViewHandler.cs#L29 
Is not working as expected here?

Reply:
@parhamsaremi hey Parham can you look into this tomorrow please?

Reply:
@jsuarezruiz  We can't seem to reproduce the problem we had with visibility anymore so I removed this commit for the PR.